rousseau_vote/pubspec.yaml

```diff
@@ -86,6 +86,8 @@ flutter:
   assets:
     - l10n/it.json
     - assets/images/
+    - assets/merits/
```

Comment:
Merits should be a subfolder of images since it's logically a subset. Under asset, by convention, there should be one folder per type.